CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Cancer case_report panel-table no longer contains inheritance information
 - Case report pinned variants card now displays info text if all pinned variants are present in causatives
 - Darkmode setting now applies to the comment-box accordion
+- Typo in case report causing `cancer_rank_options is undefined` error
```

Comment:
Perhaps state that it was fixed?

Reply:
It's already in the "Fixed" category, I think it's obvious that it's a fixed typo

Reply:
Thanks for the review, merging!